test/v3/controllers/SubmitForeignPropertyBsasControllerSpec.scala

```diff
@@ -245,17 +252,19 @@ class SubmitForeignPropertyBsasControllerSpec
         }
 
         val input = Seq(
```

Comment:
Do we not need `InvalidTaxYearParameterError` here also?

Reply:
Should we also split the Seq into a concatenation of 2 x Seq (1 x pre-Tys, 1 x Tys)?

Reply:
No because `InvalidTaxYearParameterError` will not come from the downstream service.

Didn't split them up because the controller is not directly associated with the downstream error codes added with TYS.